languageFactories/language-factory-core/src/test/scala/cromwell/languages/util/ImportResolverSpec.scala

```diff
@@ -34,14 +36,14 @@ class ImportResolverSpec extends FlatSpec with Matchers {
   }
 
   it should "resolve a path from no initial root" in {
-    val resolver = HttpResolver()
+    val resolver = HttpResolver(emptyRootWfResolvedImports)
```

Comment:
It would be nice to verify that we're getting the expected set of imports added to `ResolvedImports` during all these tests.

Reply:
I tried adding asserts for these tests. But since each test uses a fake path they failed to resolve by the resolvers and threw errors. The `WomtoolValidateSpec` does indirectly cover that the right set of imports are added to `ResolvedImportsStore`.